### DISCUSSION TITLE
用外部PDF阅读器看的条目无法在最近在读中显示

### DISCUSSION THREAD

**Discussion by awkar** [volatile-static/Chartero#125] [General]
如题。如果一个条目我是用外部PDF阅读器打开的话，无法在最近阅读中显示，也无法在总览中看到阅读时间。
我想问一下这个问题是不是无解？毕竟跨应用了。
此外，我用外部PDF阅读器是因为我习惯用文字注释，而自带的reader输入文字注释需要分别调整字体大小，默认的字体太大了，也不能调整字体。最后注释也不能直接保存在PDF文件中。请问有什么插件或方法修改自带reader的文字注释（即新增文字那个）的默认字体大小，不需要每次都要重新调整；然后怎么把注释直接保存到PDF文件中。
我用的是zotero 7.0.2

**Comment by volatile-static** [comment]
是的，你提到这些需求我也认为是应该实现的，但我目前确实没见过有这样的插件
